file/file.go

```diff
@@ -132,8 +132,30 @@ func NewSubscriber(root string) bps.Subscriber {
 	}
 }
 
-func (s *fileSub) Subscribe(ctx context.Context, topic string, handler bps.Handler, _ ...bps.SubOption) error {
-	f, err := os.Open(filepath.Join(s.root, topic))
+func (s *fileSub) Topic(name string) bps.SubTopic {
+	return NewSubTopic(filepath.Join(s.root, name))
+}
+
+func (s *fileSub) Close() error {
+	return nil
+}
+
+// ----------------------------------------------------------------------------
+
+type subTopic struct {
```

Comment:
why not just `type subTopic string`

Reply:
`type SubTopic string` then to get rid of constructor